CHANGELOG.md

```diff
@@ -5,6 +5,39 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## Release 5.0.0 (2020-09-22)
+
+[Changes since v4.0.0](https://github.com/realm/realm-studio/compare/v4.0.0...v5.0.0)
+
+### Breaking Changes
+
+- Added support for reading / writing Realm files using Realm file format v11 (used by realm-js v6.1.1, realm-java v7.0.4, realm-cocoa v5.4.0 and realm-dotnet v5.0.1). When opening a file which is using an older format, you will get prompted to upgrade (and optionally backup) the Realm file. NOTE: Once the file has been upgraded, there is no way to downgrade it again and it can only be used by the SDKs that support Realm file format v11. ([#1331](https://github.com/realm/realm-studio/pull/1331))
```

Comment:
Should we remove the patch version? Or actually just mention the version where the new fileformat was introduced?

Reply:
Should we add instructions (link?) that describes how to keep the old version of Studio in parallel with the new?
"If you have a need to keep working on both pre v11 and v11 file formats, you can install this new version along an existing version by ..."

Reply:
> Should we remove the patch version?
No, I don't think so, since those are the exact versions the new file format got introduced to the respective SDKs. The major versions introduced the **v10** file format.

> Should we add instructions (link?) that describes how to keep the old version of Studio in parallel with the new?
We don't have that page written up yet. I suggest we patch the changelog (or even better the readme.md) once we have that page ready.

Reply:
Ok, that's perfect then!